layouts/company/single.html

```diff
@@ -129,6 +129,17 @@ <h2>{{ T "cdb-related-companies" }}</h2>
                 </ul>
             </div>
         {{ end }}
+    {{ else }}
+    <div id="related-companies">
+        <h2>Related companies</h2>
+        <ul>
+            <li><a href="https://www.datarequests.org/company/quotient-com/">Quotient Technology Inc.</a></li>
```

Comment:
After some internal discussion, we think that it would be better to use obvious fake company names (something like *Acme Inc.*) here to make it clear that these are only placeholders. These should then all link to our company page. Please use `{{< ref "company/datenanfragen" >}}` to make sure the URL is currently localized.

Reply:
You could use "Darkenanfragen AG"! 😄